manifests/params.pp

```diff
@@ -34,9 +37,12 @@
       $ldap_auth_plugin_location = undef # no ldap plugin on redhat/centos
     }
     'Debian': { # Debian/Ubuntu
-      $group = 'nogroup'
-      $link_openssl_cnf = true
-      $pam_module_path = '/usr/lib/openvpn/openvpn-auth-pam.so'
+      $etc_directory     = '/etc'
+      $root_group        = 'root'
+      $group             = 'nogroup'
+      $link_openssl_cnf  = true
+      $pam_module_path   = '/usr/lib/openvpn/openvpn-auth-pam.so'
+      $load_main_service = true
```

Comment:
Due to the way things were written previously, it came in hand for disabling the 'main' Service, it's an inverted boolean option for `namespecific_rclink`, which will have a service per server.
I removed it in `f225835`